config/test.js

```diff
@@ -8,12 +8,8 @@
 
 'use strict';
 
-const os = require('os');
-const home = os.homedir();
-
 module.exports = {
   cli: false,
-  profileDir: `${home}/.mozilla-iot/test`,
```

Comment:
I removed this since it's already set in default.js and we don't need to set it a second time.